extensions/amp-carousel/0.2/test/test-e2e/helpers.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const TAG_NAME = 'amp-carousel';
+const SCROLLABLE_SLIDE = 'amp-scrollable-carousel-slide';
+const NEXT_ARROW_SELECTOR = `.amp-carousel-button.amp-carousel-button-next`;
+
+export function getSlides(controller) {
+  return controller.findElements(`${TAG_NAME} .${SCROLLABLE_SLIDE}`);
+}
+
+export async function getSlide(controller, n) {
+  const slides = await getSlides(controller);
+  return slides[n];
+}
+
+export function getNextArrow(controller) {
+  return controller.findElement(NEXT_ARROW_SELECTOR);
+}
+
+export function sleep(ms) {
+  return new Promise((res) => setTimeout(res, ms));
+}
```

Comment:
Some of these appear to be taken from the helpers for `amp-base-carousel` tests. Can they be shared instead?

Reply:
They are (and same w/ the images), but I wasn't sure if cross extension dependencies (even if its just in tests) is good practice. WDYT?